### PR TITLE
Fix version and package naming

### PR DESCRIPTION
**Pull request by adybbroe**
Change package name to nwcsafpps_runner, fix for Python3 and git versioning.

### CODE REVIEW

File: nwcsafpps_runner/utils.py [hunk 1]
```diff
@@ -606,7 +606,7 @@ def publish_pps_files(input_msg, publish_q, scene, result_files, **kwargs):
                          '/' + station + '/' + environment +
                          '/polar/direct_readout/',
                          "file", to_send).encode()
-        LOG.debug("sending: " + str(pubmsg))
+        #LOG.debug("sending: " + str(pubmsg))
```
Comment (mraspaud) [inline nwcsafpps_runner/utils.py:609]: Please remove it instead of commenting it.

**Review by mraspaud — CHANGES_REQUESTED** [review]
Some inline comments. Also I do agree with some of the codefactor.io comments.

**Review by TAlonglong — COMMENTED** [review]
>>> n = datetime.datetime.now()
>>> n.strftime("%Y%m%dT%H%M%S")
'20200928T143910'

**Review by TAlonglong — COMMENTED** [review]
In additions to @mraspaud and some of the codecov this LGTM

**Comment by adybbroe** [comment]
@mraspaud and @TAlonglong Many thanks for the useful comments. 
Code should be slightly more readable now.

**Comment by adybbroe** [comment]
And, by the way, I looked at the Deep Code bot's comments and found it partly useful only. Fixed one issue. But line numbers doesn't point correctly...

**Comment by adybbroe** [comment]
Thanks for your reviewing @mraspaud and @TAlonglong , do you think it looks better and ok now?

**Comment by mraspaud** [comment]
@adybbroe I still think the codefactor items should be resolved

**Comment by adybbroe** [comment]
Thanks @adybbroe 
Overlooked those. I have fixed all of them but 2 I believe. One I have asked your advice on, the other is the FIXME! which I don't have an answer to yet.

**Comment by adybbroe** [comment]
Okay, now down to 3 issues. The two I mentioned above plus a third on "Complex code" on line 0 in `utils.py`
No idea atm what to do about that...

**Comment by mraspaud** [comment]
Complex code can be helps with refactoring

**Comment by adybbroe** [comment]
> Complex code can be helps with refactoring

Ok, but there was no hint. 
I can certainly look over the complexity and refactor, but not in this  PR!

**Comment by adybbroe** [comment]
Ok to merge @mraspaud and @pnuu ?

**Comment by mraspaud** [comment]
I don't see any tests, but I guess it's the rule in this package? If you are ok with that, then you can merge imo.

**Comment by adybbroe** [comment]
> I don't see any tests, but I guess it's the rule in this package? If you are ok with that, then you can merge imo.

Yes, it is unfortunately the rule in this package. I can make an issue on that

**Pull request merged**